.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18.x'
+      - name: Print versions
+        run: |
+          node --version
+          npm --version
+          npx commitlint --version
```

Comment:
```suggestion
```
I don't think we need this step in a live pipeline, can be added back anytime for debugging and takes too long to execute (11s). Also: `npx commitlint --version` will always fail since it's only being installed in the next step.